### PR TITLE
pgcli: update to 2.1.0 and fix dependencies

### PR DESCRIPTION
**Pull request by blazewicz**
#### Description

Upstream pgcli does not support psycopg2 version above 2.8.0 since https://github.com/dbcli/pgcli/pull/859 because it requires build tools and libraries to compile instead of shipping compilled stuff in wheel.

Because macports already compile and install psycopg2 in version 2.8.2 I think its best choice is to simply apply patch to pgcli's dependencies.

Also updated py-cli-helpers as its required in newest version of pgcli.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.4 18E226
Xcode 10.2.1 10E1001

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: databases/pgcli/Portfile [hunk 1]
```diff
@@ -17,9 +17,10 @@ long_description    ${description}
 
 homepage            https://pgcli.com
 
-checksums           rmd160  1d508c4b6232928b96e6f2961b76fc2d7d332e10 \
-                    sha256  e3264edfd37fce23cb8b007888e01b0eb2fe537d6938ce660e82fdc109d764b5 \
-                    size    432372
+checksums           rmd160  5766d6c41d8dde41e4a7ef7e787d0dce446838ec \
+                    sha256  1ba3b23267e0f9cc361573f61d661792302d2d012ac17c597efa52124075716b \
+                    size    433647
+
```
Comment (xeron) [inline databases/pgcli/Portfile]: nitpick: excess empty line.

Comment (blazewicz) [reply databases/pgcli/Portfile]: true, forgot to remove that!

**Comment by mf2k** [comment]
It's a nitpick, but the commit message should mention the update of the py-cli-helpers port too.

**Comment by reneeotten** [comment]
> It's a nitpick, but the commit message should mention the update of the py-cli-helpers port too.

I agree, this should ideally have been two commits (or at least have mentioned it in the commit messages explicitly) ... @blazewicz can you address this so it can get merged (alternatively @mf2k would it really be blocking a merge or could we just say "please do this differently next time")?

**Comment by blazewicz** [comment]
@reneeotten done

I also explained the reason for the patch in commit message.

**Pull request merged**